how-we-work/tracking-time.md

```diff
@@ -1,116 +1,114 @@
+A special mention goes to the **Time Wasting** project. This is a generic project where you can
+track any time you have spent at the office, but not actually working. For instance, this could
+be time you have spent having coffee with a colleague, time you spent at lunch or anything else that
+does not strictly qualify as work but still happened _at_ work.
 
-- time of direct service or implementation;
-- time spent pairing on the project;
-- communication time directly related to the work performed;
-- study time directly related to the work performed.
+Usage of this project is optional. When it is available, we use the data we collect from the project
+to see if there are any inefficiencies in company organization that we can streamline. However, we
+do not want to invade people's privacy, so feel free not to use it if you are not comfortable with
+sharing how you spend this time or if you simply find it too cumbersome to start the time tracker in
+these situations.    
 
-On the other hand, the tracker should be switched to another project for:
+## Presence vs. work
 
-- time of direct service or implementation on other projects;
-- time spent pairing with another person on other projects;
-- communication time not directly related to the work performed;
-- study time not directly related to the work performed.
+You may have noticed there is some overlap between the two concepts of "hours of presence" and 
+"hours of work".
 
-> A particular case: Should I include the time spent waiting for client's responses, when I am
-> twiddling my thumbs? The answer is: never be in a situation where you have to twiddle your thumbs
-> and find something to do anyway. If you really have nothing to do, this time is NOT included in the
-> "time actually spent", but the client should be informed promptly, reporting the inefficiency.
+For instance, if you have reported 9 hours of work on Harvest, it's clear you have worked overtime,
+no matter how you have spent those hours (even if one hour was spent on Time Wasting, it's still an
+hour spent at work).
 
-There is no need to start the tracker for break times, but we leave the freedom, to those who want,
-to assign this time to the *Time wasting* project.
+On the other hand, if you have reported 7 hours, you have clearly taken some time off (again, you
+should sum any Time Wasting to your reported hours).
 
-Some difficulties may arise especially when faced with having to deal with multiple activities
-at the same time. We recommend you avoid multitasking. Focusing on homogeneous tasks allows you to
-work under less stress with higher productivity and makes using the tracker much easier.
+With that said, you should not look at Harvest too closely to determine your hours of presence. If
+you have tracked 7:30 hours but still feel like you have worked a full day, there is no need to
+report 30 minutes of time off, because you've still only spent 8 hours at the office.
```

Comment:
> For instance [...] office.

I'm not convinced about the whole section. We should clarify, regardless of the Time wasting project, that no matter how many hours you have on the time-tracker, you have to compute your hours of presence to understand if you have to report overtime or time off on the presence sheet.

IMO, two useful contrasting examples could be:
1)  I have 8 hours tracked at the end of the day, and probably I took from 4 to 6 pause times (**not tracked** with the Relax task). Your hours of presence in the office **are certainly more than eight** so you've to compute and report overtime.
2 ) I have 6,5 hours tracked at the end of the day, and probably I took from 4 to 6 pause time (**not tracked** with the relax task). Your hours of presence in the office **are not certainly less than eight** so you've to compute starting and finishing time in the office or in your remote working space and reporting time off or nothing depending on the result.

Reply:
@matteogalliani hmm, I'm not sure how what you wrote differs from what is in that section. The concept I wanted to convey is that tracked hours can be used as a measure of whether you've done overtime or took time off, but you should still calculate any overtime/time off manually. You think that's not clear?